src/main/java/hudson/plugins/copyartifact/DownstreamBuildSelector.java

```diff
@@ -196,7 +196,7 @@ public FormValidation doCheckUpstreamProjectName(
             }
             
             AbstractProject<?,?> upstreamProject = jenkins.getItem(
-                    upstreamProjectName, project.getRootProject(), AbstractProject.class
+                    upstreamProjectName, project, AbstractProject.class

```

Comment:
You might misunderstand the comment by jglick (https://github.com/jenkinsci/copyartifact-plugin/pull/79/files/65b30d6da1983e11be0f93d9defb664796fca99d#r56505288)
That meant you don't need to use `null` for `project` even when `project` doesn't have `getRootProject`.
`getRootProject` should be still required if it's available.

Though I don't know an actual case resulting `project.getRootProject() != project` in this context, it's better to implement in the same way as `DownstreamBuildSelector` works.